src/corelibs/U2Gui/src/ObjectViewModel.h

```diff
@@ -353,24 +366,25 @@ class U2GUI_EXPORT GObjectViewAction : public QAction {
     int actionOrder;
 };
 
+/**
+ * Abstraction of the object handler for a view.
```

Comment:
It's not "abstraction" now, but you could make it abstract by marking function as pure virtual

Reply:
It is abstraction. Abstract classes/interfaces can have default impls.

Reply:
Strongly say, abstract class in C++ is a class that has at least one pure virtual function, interfaces are not presented in C++ at all, but we can define them as "abstract classes where the all functions are pure virtual". This class doesn't have pure virtual functions at all, so it's just the base class with the overridable content. Bit it's about word choice, I won't insist.